### PR TITLE
docs: getting started  with vs code section

### PR DESCRIPTION
**Pull request by debs-obrien**

### CODE REVIEW

File: docs/src/getting-started-vscode-js.md [hunk 1]
```diff
@@ -83,16 +83,28 @@ test('test', async ({ page }) => {
 
 ## Running the Tests
 
+You can run a single test, all tests in a file or all tests in the tests folder. Tests can be run on one browser or multiple browser.
```
Comment (rwoll) [inline docs/src/getting-started-vscode-js.md:86]:
```suggestion
You can run a single test, all tests in a file or all tests in the tests folder. Tests can be run on one browser or multiple browsers.
```

File: docs/src/getting-started-vscode-js.md [hunk 2]
```diff
@@ -83,16 +83,28 @@ test('test', async ({ page }) => {
...
+
+In the sidebar you can extend the tests by clicking on it. This will show you the tests for your generated test as well as the example test that came when installing Playwright. This test has not been run therefore it is not in green. 
 
 
 <img width="812" alt="image" src="https://user-images.githubusercontent.com/13063165/177201231-f26e11da-2860-43fa-9a31-b04bba55d52e.png" />
 
-You can rerun a single test or all tests by clicking on the triangle as you hover over each of the tests in the sidebar.
+
+### Running All Tests
+You can all tests by clicking on the white triangle as you hover over the tests in the sidebar.
```
Comment (rwoll) [inline docs/src/getting-started-vscode-js.md:105]:
```suggestion
You can run all tests by clicking on the white triangle as you hover over the tests in the sidebar.
```

File: docs/src/getting-started-vscode-js.md [hunk 3]
```diff
@@ -130,16 +144,20 @@ To remove the breakpoint click on the red dot next to line 9 so it disappears.
...
 
-Once the tests have finished run the folling command to open the reporter.
+Once the tests have finished run the following command to open the reporter.
 
 ```bash
-npx playright report
+npx playright show-report
 ```
 
+You can then filter the report by browers, failed tests, skipped tests etc.
```
Comment (rwoll) [inline docs/src/getting-started-vscode-js.md:157]:
```suggestion
You can then filter the report by browsers, failed tests, skipped tests etc.
```

**Review by rwoll — APPROVED** [review]
Modulo the comments, LGTM

File: docs/src/vs-code-extension-js.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+---
+id: vs-code-extension
```
Comment (mxschmitt) [inline docs/src/vs-code-extension-js.md:2]: I'd maybe do vscode-extension. But thats a nit!

**Review by mxschmitt — APPROVED** [review]

**Pull request merged**